guide/admin/tags.md

```diff
@@ -0,0 +1,57 @@
+
+![Administration Link Screenshot](687474703a2f2f692e696d6775722e636f6d2f575a315a6a78392e676966.gif)
+
+Once you are in the Administration Panel, you'll see some navigation tabs flowing down the left hand side, click on **Tags** to reveal the configuration page to the right hand side.
+
+![Tags Screenshot](687474703a2f2f692e696d6775722e636f6d2f354f58595454642e706e67.png)
+
+### Creating a new Tag
+
+A tag is a way of categorising posts made by a user, they are primarily set by the staff of the forum (admins).
+
+On the configuration page for **Tags** you'll need to click **Create Tag** which will bring up the create tag box:
+
+![SS Tag Creation Box](687474703a2f2f692e696d6775722e636f6d2f304772355739742e706e67.png)
+
+### Create Tag Components
+
+#### Name:
+
+The name is what you'd like the tag to be called, also known as the category title.
+
+#### Slug:
+
+The slug is a string that goes into the URL, essentially it's a version of the name of the tag that is all lower-case, and uses dashes instead of spaces.
+
+Unless you need to, you generally don't need to do anything with this, as when you type in the **Name** the slug auto-fills in.
+
+#### Description:
+
+The description of the tag will appear alongside the **name** in the tag selection post when making a post, it's a short few words giving a more in-depth description of the category it belongs to.
+
+#### Colour:
+
+This is the colour you would like the tag to appear in on the posts, and forums, the colour box uses **HTML Colour Codes** beginning with a **#** you can Google around for HTML Colour Codes, there are plenty of websites on this.
+
+Once you've filled in the appropriate information in the Create Tag prompt, click on **Save Changes** to add the new tag.
+
+### Moving the Tag between Primary, and Secondary tags
+
+By default, new tags created will be **Secondary Tags** also known as **Sub Categories**, sometimes you'll want these to be primary tags, to do this, click and drag the tag to the **Primary** slot at the top, follow the below animation to see how this is done:
+
+![Anim - Moving Tags](687474703a2f2f692e696d6775722e636f6d2f765738796e42732e676966.gif)
+
+### Deleting a Tag
+
+To delete a tag, hover your mouse to the right of the tag you wish to remove, and click on the **Pencil Icon** this will bring up the editing prompt, to the bottom right of that prompt, you'll see the **Delete tag** link, click that and then click **OK** to delete the tag.
+The below animation will show you how this is done:
+
+![Anim - Delete a Tag](687474703a2f2f692e696d6775722e636f6d2f635664654a7a772e676966.gif)
+
+### Editing a Tag
+
+Made a mistake, or need to change something on a tag? follow the same procedure as above in **Deleting a Tag** until you get to the editing prompt, this time, **DON'T** click on Delete Tag, instead edit the information displayed until you are happy, then click **Save Changes**. Easy as that!
```

Comment:
Re-wrote the section, merged them together (with some extra screenshots), and added part about hide from all discussions. Fixed.